internal/pkg/deploy/cloudformation/stack/worker_svc_test.go

```diff
@@ -189,6 +189,9 @@ Outputs:
 					StringSlice: []string{"here"},
 				}
 				svc.manifest.ExecuteCommand = manifest.ExecuteCommand{Enable: aws.Bool(true)}
+				svc.manifest.DeployConfig = manifest.DeploymentConfiguration{
+					Rolling: aws.String("incorrect value"),
```

Comment:
This test case looks a little weird to me where you can have "incorrect value" while still producing a default configuration 😆  Like conceptually we should error out instead.

In real life though I suppose this won't happen because of manifest validation. I think we should use either blank, or "recreate", or "default" in this test case.

Reply:
Right, make sense. Manifest validation will never let this kind of inputs go further. So this use case doesn't make sense. I will change it to have "default" value.